pkg/bpf/bpf_linux.go

```diff
@@ -128,14 +139,19 @@ func LookupElement(fd int, key, value unsafe.Pointer) error {
 		value: uint64(uintptr(value)),
 	}
 
-	duration := spanstat.Start()
+	var duration *spanstat.SpanStat
+	if option.Config.IsSubsysMetricEnabled(metrics.SubsystemBPFMask) {
```

Comment:
This isn't blocking, but I feel like this check would be way simpler overall if the value being checked was a variable set at the package level? We can use `option.Config` to gather all configuration from daemon bootstrap, then at bootstrap of this package, take in the value from `option.Config` and set it as a package-scoped variable.

Reply:
@ianvernon That's going definitely be a follow up as the performance improves as well.

```
5506 ns/op vs 5365 ns/op
```